src/pages/play/styles.module.scss

```diff
@@ -0,0 +1,102 @@
+$grey-accent: #333333;
+
+// Reset Infima styles
+:global .monaco-editor {
+    .button {
+        background-color: inherit;
+        border: inherit;
+        box-sizing: inherit;
+        color: inherit;
+        cursor: inherit;
+        display: inherit;
+        font-size: inherit;
+        font-weight: inherit;
+        line-height: inherit;
+        outline: inherit;
+        padding: inherit;
+        text-align: inherit;
+        text-decoration: inherit;
+        transition: inherit;
+        user-select: inherit;
+        vertical-align: inherit;
+        white-space: inherit;
+    }
+
+    .contents {
+        list-style-type: inherit;
+        font-size: inherit;
+        margin-bottom: inherit;
+        padding-bottom: inherit;
+        padding-top: inherit;
+    }
+}
+
+$navbar-height: 50px;
+
+.navbar {
+    border-bottom: 1px $grey-accent solid;
```

Comment:
I don't like these hard borders, maybe remove them, replace them by a shadow or use a lighter grey.

Reply:
> use a lighter grey

How about the one used on https://www.typescriptlang.org/v2/en/play (#c4c4c4) (for a light theme, I think current color is nice for the dark one)

Reply:
LGTM